src/transformers/searchservice/createDirectAnswer.ts

```diff
@@ -1,14 +1,28 @@
-import { DirectAnswer } from '../../models/searchservice/response/DirectAnswer';
+import { FeaturedSnippetDirectAnswer, FieldValueDirectAnswer } from '../../models/searchservice/response/DirectAnswer';
+import { DirectAnswerType } from '../../models/searchservice/response/DirectAnswerType';
 import { ResultsFactory } from './ResultsFactory';
 
-export function createDirectAnswer(data: any): DirectAnswer {
-  return {
-    relatedResult: ResultsFactory.fromDirectAnswer(data.relatedItem.data),
-    verticalKey: data.relatedItem.verticalConfigId,
-    entityName: data.answer.entityName,
-    fieldName: data.answer.fieldName,
-    fieldApiName: data.answer.fieldApiName,
-    value: data.answer.value,
-    fieldType: data.answer.fieldType
-  };
+export function createDirectAnswer(data: any): FeaturedSnippetDirectAnswer | FieldValueDirectAnswer | undefined {
```

Comment:
Why do we allow `undefined` here? I would expect some indication that we didn't create a direct answer right?

Reply:
A direct answer is an optional field of a `UniversalSearchResponse`, so returning undefined is what we want to do if the backend doesn't return a direct answer. Per Tom's suggestion, I will update this function to throw an error if we get an unknown DA type